### ISSUE TITLE
Problems (red) on Java files/packages mostly after cleaning workspace

### ISSUE THREAD

**Issue by benzman81** [redhat-developer/vscode-java#2602]
I have a problem where Java files and packages  in vscode stay red showing a problem. This happens mostly after cleaning workspace.
I use workspaces with many different maven projects in each and it occures in all of them.

The problems shown in the problems view are The `import foo.bar cannot be resolved` and `MyJavaClass cannot be resolved to a type`. I can only fix this, if I open every single file once. Then the problmes go away automatically, but this is really annoying to do.

##### Environment
- Operating System: Windows 10 20H2
- JDK version: 
- Visual Studio Code version: 1.69.2
- Java extension version: 1.9.0

##### Steps To Reproduce
Clean workspace with multiple folders of multimodule maven projects. At least using my projects this happens.

##### Current Result
VS Code showing a bunch of problems and red projects

![image](https://user-images.githubusercontent.com/14224592/181741618-72b0b2c4-d3a4-4f01-9a24-a06a0a128c48.png)

##### Expected Result
No problems in VS Code and no red projects.

##### Additional Informations
No more hints i can give.

**Comment by jdneo** [comment]
Are you using lombok in your project?

**Comment by benzman81** [comment]
No, we dont. This issue also occurs on laptops of my colleagues.

**Comment by jdneo** [comment]
Not sure if it's related with this one: https://github.com/microsoft/vscode-java-pack/issues/1042

@CsCherrYY do you have any insight?

**Comment by zoily** [comment]
I may be encountering the same issue. When I clean install using maven, JSL start rebuilding during install. It throws errors in each file even if the build is successful. At the end of my maven build, I run Rebuild projects, select all. It then build perfectly. I avoid doing clean install as much as I can but it is still often necessary
